composer.json

```diff
@@ -1,29 +1,52 @@
 {
   "name": "spryker-eco/inxmail",
+  "type": "library",
   "description": "Inxmail module",
+  "license": "MIT",
   "require": {
-    "php": ">=7.1",
-    "spryker/kernel": "^3.0.0",
+    "php": ">=7.2",
+    "spryker/application": "^1.0.0 || ^2.0.0 || ^3.0.0",
+    "spryker/customer-extension": "^1.0.0",
     "spryker/guzzle": "^2.2.0",
-    "spryker/oms": "^8.0.0 || ^10.0.0 || ^11.0.0",
-    "spryker/customer": "^7.10.0",
-    "spryker/mail": "^4.0.0"
+    "spryker/kernel": "^3.0.0",
+    "spryker/locale": "^1.0.0 || ^2.0.0 || ^3.0.0",
+    "spryker/money": "^1.0.0 || ^2.0.0",
+    "spryker/product": "^5.0.0 || ^6.0.0",
+    "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0",
```

Comment:
This is kinda crazy..
Are we sure this is true? 
Did you manually download 5.0.0 and replaced the core here just to have a quick smoke test?

Reply:
There is only one usage of SalesFacade via a bridge. I think it's true. The problem that the first version of module was released without correct dependencies. I don't want to use the last versions of the dependencies to not force customer projects to update other modules.

Reply:
I think it should work for all the versions correctly.

/**
     * @param int $idSalesOrder
     *
     * @return \Generated\Shared\Transfer\OrderTransfer
     */
    public function getOrderByIdSalesOrder($idSalesOrder);